package.json

```diff
@@ -1,15 +1,22 @@
 {
   "name": "purescript-elmish-html",
   "description": "Render HTML elements in Elmish",
-  "version": "0.1.0",
-  "private": true,
-  "author": "Canopy Education, Inc.",
+  "version": "0.2.0",
+  "author": {
+    "name": "Daniel Gasienica",
+    "email": "daniel@gasienica.ch",
+    "url": "https://www.gasi.ch"
+  },
   "contributors": [
-    "Daniel Gasienica <daniel@gasienica.ch>"
+    {
+      "name": "Fyodor Soikin",
+      "email": "fyodor@collegevine.com",
+      "url": "https://github.com/fsoikin"
+    }
```

Comment:
Note here: I changed these fields to conform them to the official `package.json` schema and used your details from https://github.com/collegevine/purescript-elmish/pull/26 @gasi